.github/workflows/inspektor-gadget.yml

```diff
@@ -1,6 +1,8 @@
 name: Inspektor Gadget CI
 env:
-  CONTAINER_REPO: ghcr.io/${{ github.repository }}
+  REGISTRY: ghcr.io
+  RELEASE_CONTAINER_REPO: ${{ github.repository }}
```

Comment:
Did you verify pushing to main or a tag in a private repo uses the correct repo?

Reply:
Could you please rebase this PR after 740 was merged to verify this problem is gone?